listenbrainz_spark/stats/sitewide/entity.py

```diff
@@ -133,31 +96,24 @@ def get_entity_all_time(entity: str) -> Optional[List[SitewideEntityStatMessage]
     to_date = get_latest_listen_ts()
     from_date = datetime(LAST_FM_FOUNDING_YEAR, 1, 1)
 
-    # Generate a dataframe containing years from "from_date" to "to_date"
```

Comment:
This whole block was never used?

Reply:
This is the part we discussed briefly in IRC last week. The way this had been implemented till now was that if spark was requested all time sitewide stats, it would generate the top artists for each year since founding i.e. 2005 top artists, 2006 top artists, 2007 top artists .... 2021 top artists. While all time user stats simply mean the overall top artists.

Similarly, year sitewide stats for eg: of 2021 meant top artists of Jan 2021, top artists of Feb 2021 and so on. While year user stats mean the overall top artists of the year.